internal/cluster/validations/validations.go

```diff
@@ -506,9 +525,20 @@ func ValidateClusterUpdateVIPAddresses(ipV6Supported bool, cluster *common.Clust
 	targetConfiguration.UserManagedNetworking = swag.Bool(false)
 	if params.UserManagedNetworking != nil {
 		if swag.BoolValue(params.UserManagedNetworking) {
-			err = ValidateVIPsWereNotSetUserManagedNetworking(cluster.APIVip, cluster.IngressVip,
-				cluster.APIVips, cluster.IngressVips, swag.BoolValue(cluster.VipDhcpAllocation))
-			if err != nil {
+
+			if swag.BoolValue(cluster.VipDhcpAllocation) { // override VIPs that were allocated via DHCP
```

Comment:
suggesting to move it below `if err = validateVIPsWereNotSetUserManagedNetworking(cluster, vipDhcpAllocation); err != nil {` 
here it looks out of context
reading the code wasn't clear to me.

Reply:
Done